arm-asazure/2016-05-16/swagger/asazure.json

```diff
@@ -192,6 +192,78 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers/{serverName}/suspend": {
+      "post": {
+        "description": "Supends the specified Analysis Services server instance",
+        "tags": [ "Servers" ],
+        "operationId": "Servers_Suspend",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "serverName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z][a-z0-9]*$",
+            "minLength": 3,
+            "maxLength": 63,
+            "description": "Name of the Analysis Services server"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
```

Comment:
You use `202 Accepted` -- is this supposed to be a long-running operation ( ie, use `"x-ms-long-running-operation": true` ?)